layouts/partials/intersector.html

```diff
@@ -1,10 +1,7 @@
 <span class="o-divider" aria-hidden="true">
   <hr>
-  <svg viewBox="0 0 37 24" xmlns="http://www.w3.org/2000/svg">
-    <g style="fill: var(--link-default, #D54716);" id="ticket" transform="translate(4, 4)" fill="var(--link-default)" fill-rule="nonzero">
-      <path d="M28.2926829,0 C28.6833234,0 29,0.325611091 29,0.727272727 L29,5.45454545 C27.4374383,5.45454545 26.1707317,6.75698982 26.1707317,8.36363636 C26.1707317,9.97028291 27.4374383,11.2727273 29,11.2727273 L29,15.2727273 C29,15.6743889 28.6833234,16 28.2926829,16 L0.707317073,16 C0.31667664,16 0,15.6743889 0,15.2727273 L0,11.2727273 C1.56256173,11.2727273 2.82926829,9.97028291 2.82926829,8.36363636 C2.82926829,6.75698982 1.56256173,5.45454545 0,5.45454545 L0,0.727272727 C0,0.325611091 0.31667664,0 0.707317073,0 L28.2926829,0 Z M26.8780488,2.18181818 L2.12195122,2.18181818 L2.12195122,3.76290909 L2.16408965,3.78347584 C3.78297735,4.59366299 4.90812712,6.28454086 4.95000928,8.24994726 L4.95121951,8.36363636 C4.95121951,10.3773724 3.8141098,12.1180292 2.16408965,12.9437969 L2.12195122,12.964 L2.12195122,13.8181818 L26.8780488,13.8181818 L26.8780488,12.9643636 L26.8359104,12.9437969 C25.2170227,12.1336097 24.0918729,10.4427319 24.0499907,8.47732546 L24.0487805,8.36363636 C24.0487805,6.34990032 25.1858902,4.60924351 26.8359104,3.78347584 L26.8780488,3.76254545 L26.8780488,2.18181818 Z" id="Combined-Shape"></path>
-      <path d="M20.5121951,11.6363636 C21.0981558,11.6363636 21.5731707,12.1247803 21.5731707,12.7272727 L21.5731707,14.5454545 C21.5731707,15.147947 21.0981558,15.6363636 20.5121951,15.6363636 C19.9262345,15.6363636 19.4512195,15.147947 19.4512195,14.5454545 L19.4512195,12.7272727 C19.4512195,12.1247803 19.9262345,11.6363636 20.5121951,11.6363636 Z M20.5121951,5.81818182 C21.0981558,5.81818182 21.5731707,6.30659845 21.5731707,6.90909091 L21.5731707,9.81818182 C21.5731707,10.4206743 21.0981558,10.9090909 20.5121951,10.9090909 C19.9262345,10.9090909 19.4512195,10.4206743 19.4512195,9.81818182 L19.4512195,6.90909091 C19.4512195,6.30659845 19.9262345,5.81818182 20.5121951,5.81818182 Z M20.5121951,0 C21.0981558,0 21.5731707,0.488416637 21.5731707,1.09090909 L21.5731707,4 C21.5731707,4.60249245 21.0981558,5.09090909 20.5121951,5.09090909 C19.9262345,5.09090909 19.4512195,4.60249245 19.4512195,4 L19.4512195,1.09090909 C19.4512195,0.488416637 19.9262345,0 20.5121951,0 Z" id="Line"></path>
-    </g>
+  <svg>
+    <use href="{{ (resources.Get "images/intersector.svg").Permalink }}#icon" width="37" height="24"></use>
```

Comment:
```suggestion
    <use href="{{ (resources.Get "images/intersector.svg").Permalink }}"></use>
```

What does the "#icon" stand for (used for both intesector and logo)? And the width and height are already set in the styles / SCSS, so we can probably remove it here.